### DISCUSSION TITLE
Apache Tests?

### DISCUSSION THREAD

**Discussion by audouts** [TechEmpower/FrameworkBenchmarks#7495] [General]
When I try "apache-http" for the platform, there doesn't seem to be any useful data. The Nginx results also seem limited.

Am I missing something or is Apache just not really evaluated in these tests? I'd like to see how various frameworks perform on Apache and Nginx.

**Comment by joanhey** [comment]
You are correct !!
I don't want to see bad numbers about Apache. 
But nobody use it because is slow.

The next week, I'll add Apache in the 3 variants for PHP.
I did not want to do it, but you are correct.

**Comment by audouts** [comment]
Thank you. Our provider runs Apache for clients, so we don't have much of a choice. It would be nice to see a comparison. I would like to migrate away from Laravel, which seems a bit slow and bloated.
